{
  "servers": [
    {
      "name": "lambda",
      "gpus": [
        {
          "index": 0,
          "mem_used": 4,
          "mem_total": 24564,
          "util": 0,
          "processes": []
        },
        {
          "index": 1,
          "mem_used": 4,
          "mem_total": 24564,
          "util": 0,
          "processes": []
        }
      ],
      "error": null
    },
    {
      "name": "achtung-gamma",
      "gpus": [
        {
          "index": 0,
          "mem_used": 31352,
          "mem_total": 95830,
          "util": 0,
          "processes": []
        },
        {
          "index": 1,
          "mem_used": 92569,
          "mem_total": 95830,
          "util": 98,
          "processes": [
            {
              "pid": "3338428",
              "name": "python",
              "user": "shuyilin",
              "mem": 518
            },
            {
              "pid": "3400520",
              "name": "VLLM::EngineCore",
              "user": "shuyilin",
              "mem": 92036
            }
          ]
        },
        {
          "index": 2,
          "mem_used": 58112,
          "mem_total": 95830,
          "util": 61,
          "processes": [
            {
              "pid": "3936508",
              "name": "python",
              "user": "saefurukawa",
              "mem": 58102
            }
          ]
        },
        {
          "index": 3,
          "mem_used": 85974,
          "mem_total": 95830,
          "util": 0,
          "processes": [
            {
              "pid": "3989839",
              "name": "/home/aglc/rfcllm/myvenv/bin/python3",
              "user": "aglc",
              "mem": 85964
            }
          ]
        }
      ],
      "error": null
    },
    {
      "name": "achtung-biggpu",
      "gpus": [
        {
          "index": 0,
          "mem_used": 5,
          "mem_total": 49140,
          "util": 0,
          "processes": []
        },
        {
          "index": 1,
          "mem_used": 42626,
          "mem_total": 49140,
          "util": 47,
          "processes": [
            {
              "pid": "4045626",
              "name": "python",
              "user": "luzsun",
              "mem": 42600
            }
          ]
        }
      ],
      "error": null
    }
  ],
  "last_updated": "2025-10-24T17:13:01.063152Z"
}